### ISSUE TITLE
how-the-sharepoint-migration-tool-works

### ISSUE THREAD

**Issue by mozzis** [MicrosoftDocs/OfficeDocs-SharePoint#283]
This article is in really bad shape. There are many non-sequiturs, grammatical errors, and possibly irrelevant technical information. The article also appears to be missing technical information needed to understand the topic. All in all, it is really incomprehensible.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cac22961-da69-7b22-6dff-f7bc5ee82ed3
* Version Independent ID: dd4ed11e-a0f9-4e97-48ae-8bd94f510c07
* Content: [How the SharePoint Migration Tool works](https://docs.microsoft.com/en-us/sharepointmigration/how-the-sharepoint-migration-tool-works)
* Content Source: [migration/how-the-sharepoint-migration-tool-works.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/migration/how-the-sharepoint-migration-tool-works.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by AndreaBarr** [comment]
@mozzis Thank you for this feedback. 

We apricate you pointing this out. I agree that we need to work on finishing, updating, and fixing this article. It does have a lot of issues and incomplete information. 

I was able to get this into the backlog for the SharePoint team. I am going to close this issue so we don't have two tickets open for this. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
